applications/models.py

```diff
@@ -83,6 +84,44 @@ class Organisation(models.Model):
     )
 
 
+class ApplicationRound(models.Model):
```

Comment:
Round is a bit finglish, maybe something like ApplicationPeriod or maybe ApplicationCycle if you want to preserve something close to "kierros"

Reply:
Yeah, this was initially ApplicationPeriod but I changed it to ApplicationRound to include kierros in the name so there would be similarity in discussions and code concepts. I didn't think that ApplicationRound is too stupid sounding but if you think so I can change it back to ApplicationPeriod for example.

Reply:
Yeah, if you want to include kierros, then i think Cycle is closer to the Finnish meaning of kierros in this context than round.

Reply:
But i'll approve this, change it you want. Can be changed later if it bothers people.

Reply:
I changed it to application period. Now merging.